src/Symfony/Component/Mailer/Tests/EnvelopeTest.php

```diff
@@ -57,19 +66,27 @@ public function testSenderFromHeaders()
         $headers->addPathHeader('Return-Path', new Address('return@symfony.com', 'return'));
         $headers->addMailboxListHeader('To', ['from@symfony.com']);
         $e = Envelope::create(new Message($headers));
-        $this->assertEquals(new Address('return@symfony.com', 'return'), $e->getSender());
```

Comment:
@fabpot here is a slight change in behavior now since we use `parent::setAddress` and it will strip away the name of the address